docs/adr/0015-cluster-resources-amendment.md

```diff
@@ -0,0 +1,76 @@
+# Declarative Definitions for Cluster Scoped Resources
+
+- Status: accepted
+- Deciders: larsks, HumairAK, tumido
+
+Technical Story: [issue-1][1]
+
+## Context and Problem Statement
+
+In [ADR-0015][2] we laid out a directory structure of organizing cluster scoped resources. For the most part this has worked out relatively well. However, one thing that was not factored into this structure were the possibilities of name conflicts. Consider the following scenario:
+
+Suppose we have:
+1) A resource of kind Project named example-project with an apiversion: project.openshift.io/v1
+2) A resource of kind Project named example-project with an apiversion: config.openshift.io/v1
+
+Note that their API group differs, but everything else is the same. Both of these resources can exist simultaneously on a live cluster. However, these cannot co-exist in our github repo at `operate-first/apps/cluster-scope`, if we are to use the structuring as defined in [ADR-0015][2]. This is because, using the current structure, they would both need to be stored under `operate-first/apps/cluster-scope/base/projects/example-project/project.yaml`.
+
+## Considered Options
+
+The general consensus has been to keep the current structure and amend it to factor in the API Group. Some suggestions are:
+
+1. Use the following schema: `cluster-scope/base/<API_GROUP>-<KIND_PLURAL>/<METADATA_NAME>/<KIND_SINGULAR>.yaml`
+  This would factor in the API group as part of the name, and would not require additional nesting of directories
+2. Use the following schema: `cluster-scoped-resources/base/<API_GROUP>/<KIND_PLURAL>/<METADATA_NAME>/<KIND_SINGULAR>.yaml`
+  This would require additional nesting of directories, but this method has been observed in other tooling (i.e. how must-gather does their organization)
+
+Both solutions will require a `kustomization.yaml` at the base directory so that each resource may be included within an overlay.
```

Comment:
```suggestion
Both solutions will require a `kustomization.yaml` at the base directory so that each resource may be included within an overlay without modifying the default `kustomize build` behavior. 
```